packages/@adobe/spectrum-css-temp/components/button/skin.css

```diff
@@ -420,6 +420,13 @@ governing permissions and limitations under the License.
   }
 }
 
+@media (forced-colors: active) {
```

Comment:
I tried to just update the custom property value for HCM, but there were a couple of problems, so I needed to add a whole rule inside the media query.

Reply:
any reason this is in a separate media block from the other forced-colors block just above it?

Just notes for a future discussion of disabled buttons in HCM - I don't think it needs to be addressed in this PR:
I think all buttons with outline should get the green ring, floating green text isn't all that helpful either.

Then we wouldn't be dependent on the isPending state either, so it'd simplify this little bit

Reply:
I just wanted it to be separated from the variable settings block. Could go either way.